lambda_layers/core_layer/python/core_layer/handler/item_handler.py

```diff
@@ -183,3 +186,43 @@ def get_closed_items_by_url(url, session) -> List[Item]:
     items = session.query(Item).filter(Item.status == "closed").join(Item.urls).join(
         ItemURL.url).filter_by(url=url).all()
     return items
+
+
+def get_items_to_calculate_warning_tags(session) -> List[Item]:
+    items = session.query(Item).filter(
+        Item.warning_tags_calculated != True).all()
+    return items
```

Comment:
Die Methode wird doch eigentlich nur beim Update einmalig benötigt, oder?
Wäre es dann nicht sinnvoller sie in die Alembic Erweiterung zu packen?

Reply:
Hast recht. Erledigt!